### ISSUE TITLE
Microsoft Devices multiple detections

### ISSUE THREAD

**Issue by shwarmy** [MSEndpointMgr/ModernDriverManagement#24]
- WARNING: Computer detection method is currently 'SystemSKU', and multiple packages have been matched but with different SystemSKU value
 - WARNING: This should not be a possible scenario, please reach out to the developers of this script

The error log told me to reach out to you.
[ApplyDriverPackage.log](https://github.com/MSEndpointMgr/ModernDriverManagement/files/5104758/ApplyDriverPackage.log)

This occurred with new admin service script. I am testing with the legacy and will report back on that result.

**Comment by jmcwatty** [comment]
Please try this version to see if it resolves your issue.  https://github.com/jmcwatty/ModernDriverManagement/blob/master/Invoke-CMApplyDriverPackage.ps1
It has resolved the issues I was having with Surface devices

**Comment by NickolajA** [comment]
Just a question, why have you not instead of creating your own file with fixes, added those as PR? One of your fixes for spaces in the SKU string is in the original script now. As for the others, more details would be appreciated.

**Comment by NickolajA** [comment]
@shwarmy could you please provide a bit more details? I've seen this reported once before, but not been able to find the root cause yet.

**Comment by NickolajA** [comment]
@jmcwatty sorry didn't see your PR... 😆

**Comment by jmcwatty** [comment]
@NickolajA yeah it was my first time doing a pull request on github.  Wasn't sure how to link it to this issue

**Comment by NickolajA** [comment]
No worries, it's all pretty new to me as well to be frank. I'll look it over and incorporate some of it, as some parts are already fixed.

**Comment by andrewblunt** [comment]
I'm seeing this issue at the moment with a Surface Pro 6 having just made the switch to version 4 of the script. The driver pack comment is "(Models included:Surface_Pro_6_1796_Consumer, Surface_Pro_6_1796_Commercial)". 

There are no duplicate driver packages for that model. A couple of examples of what the script finds as a match are:

Microsoft Surface Laptop 2: 
[ApplyDriverPackage.log](https://github.com/MSEndpointMgr/ModernDriverManagement/files/5332364/ApplyDriverPackage.log)
(Models included:Surface_Go_1824_Consumer, Surface_Go_1824_Commercial)".

Microsoft Surface Book: (Models included:Surface_Book_2_1832, Surface_Book_2_1793)

**Comment by virtualtech516** [comment]
Same issue here with Surface Pro 7's
[ApplyDriverPackage.log](https://github.com/MSEndpointMgr/ModernDriverManagement/files/5385848/ApplyDriverPackage.log)
